### PR TITLE
added Nepali courses- php and javascript

### PR DESCRIPTION
**Pull request by parikshitadhikari**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ne.md [hunk 1]
```diff
@@ -43,3 +45,7 @@
 
 * [WordPress Complete Tutorial In Nepali](https://www.youtube.com/playlist?list=PL2OJkQtHPRie2xyBApANdVp_LUz4v7xIG) - Technology Channel
 
+
+#### PHP
+
+* [PHP Basic Tutorial In Nepali](https://youtu.be/Ec8bwfCNhkA?si=WgnbspeofHlg65VE) - Technology Channel
```
Comment (sohansai) [inline courses/free-courses-ne.md:52]: Missing newline character at end of file

Comment (parikshitadhikari) [reply courses/free-courses-ne.md]: its done now

File: courses/free-courses-ne.md [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
 * [C](#c)
 * [WordPress](#wordpress)
+* [PHP](#php)
 
 
 ### JavaScript
 
 * [JavaScript](https://www.youtube.com/playlist?list=PLckS_N3kOwFH-GCqCd6i-vPo-Z75DcOnc) - EverydayKarma 🇳🇵
 
+* [JavaScript Basic Tutorial In Nepali](https://youtu.be/_4e_Jyjmudg?si=MnMhuhWOzzXiGWIi) - Technology Channel
```
Comment (eshellman) [inline courses/free-courses-ne.md:17]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links!

File: courses/free-courses-ne.md [hunk 3]
```diff
@@ -43,3 +46,8 @@
 
 * [WordPress Complete Tutorial In Nepali](https://www.youtube.com/playlist?list=PL2OJkQtHPRie2xyBApANdVp_LUz4v7xIG) - Technology Channel
 
+
+#### PHP
+
+* [PHP Basic Tutorial In Nepali](https://youtu.be/Ec8bwfCNhkA?si=WgnbspeofHlg65VE) - Technology Channel
```
Comment (eshellman) [inline courses/free-courses-ne.md:52]: no shortened links

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-ne.md (hunk 3 above)
Comment (parikshitadhikari) [reply courses/free-courses-ne.md]: done. now have included a complete playlist for web dev, and excluded shortened links

File: courses/free-courses-ne.md [hunk 4]
```diff
@@ -7,6 +7,7 @@
 * [SQL](#sql)
 * [C](#c)
 * [WordPress](#wordpress)
+* [WebDevelopment](#webdevelopment)
```
Comment (eshellman) [inline courses/free-courses-ne.md:10]:
```suggestion
* [Web Development](#web_development)
```

File: courses/free-courses-ne.md [hunk 5]
```diff
@@ -43,3 +44,8 @@
 
 * [WordPress Complete Tutorial In Nepali](https://www.youtube.com/playlist?list=PL2OJkQtHPRie2xyBApANdVp_LUz4v7xIG) - Technology Channel
 
+
+#### WebDevelopment
```
Comment (eshellman) [inline courses/free-courses-ne.md:48]:
```suggestion
### Web Development
```

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-ne.md (hunk 5 above)
Comment (parikshitadhikari) [reply courses/free-courses-ne.md]: it has now been fixed

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**